micro-rdk/src/native/entry.rs

```diff
@@ -129,6 +130,7 @@ pub async fn serve_web_inner(
     let mut srv = ViamServerBuilder::new(mdns, cloned_exec, client_connector, app_config, 3)
         .with_http2(tls_listener, 12346)
         .with_webrtc(webrtc)
+        .with_periodic_app_client_task(Box::new(RestartMonitor::new(|| std::process::exit(0))))
```

Comment:
Calling `exit` here isn't fully faithful to the intent of restart. We could try to get fancy and `exec` `/proc/self/exe` or something, but short of that I think just `exit`ing is best.